lib/y2snapper_common.pm

```diff
@@ -104,30 +104,16 @@ sub y2snapper_clean_and_quit {
     # Ensure yast2-snapper is not busy anymore
     wait_still_screen;
     # C'l'ose the snapper module
-    send_key "alt-l";
+    wait_screen_change { send_key "alt-l"; };
 
-    my $ret;
     if ($ncurses) {
-        $ret = wait_serial("yast2-snapper-status-0", 30);
+        wait_serial("yast2-snapper-status-0", 180);
     }
     else {
         # Wait until root gnome terminal is focussed, delete the directory and close window
-        $ret = check_screen('root-gnome-terminal');
+        assert_screen('root-gnome-terminal', timeout => 180);
     }
 
-    if (!$ret) {
-        $self->{mute_post_fail} = 1;
-        record_soft_failure 'bsc#1032831';
-        $self->y2snapper_failure_analysis;
```

Comment:
Yep, y2snapper_failure_analysis is already in post_fail_hook and I believe it should stay in order to simplify analysis if test fails.